app/workers/hard_worker.rb

```diff
@@ -2,7 +2,6 @@ class HardWorker
   include Sidekiq::Worker
 
   def perform
-    # Target.where("created_at < ?", 7.days.ago)
-    Topic.create(name: 'job')
+    Target.where('created_at < ?', 7.days.ago).destroy_all
```

Comment:
I think that this this PR has 2 features included, the _Add user avatar_ and _Add week-old target deletion job_. Keep this in mind to separate the PR's accordingly in the future.

Reply:
Yes, this PR has both of them, was a mistake that I did not create this branch from development. I will try to keep that in mind for the next PR.

Reply:
You can switch the branch base of this PR